packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -71,6 +75,36 @@ class ProgressivePaginator extends React.Component {
     }
   }
 
+  getPageLabel(totalPages) {
```

Comment:
I'd recommend moving this function into a separate file and sharing it between the two implementations. It's complicated enough to want to avoid dual maintenance.

You'd also be able to write up a couple quick jest tests that way.

Reply:
Updated here https://github.com/cerner/terra-core/pull/3470/commits/961cceab67941ee54b671335a5a65987511c7b90